samples/net/syslog_net/runtime-srv-addr.conf

```diff
@@ -0,0 +1 @@
+CONFIG_LOG_BACKEND_NET_SERVER=""
```

Comment:
The overlay file should also disable ` CONFIG_LOG_BACKEND_NET_AUTOSTART=n` if I'm not mistaken, the runtime address setting will only take place if autostart is disabled.

Also regarding the filename, please use `overlay-runtime-srv-addr.conf` so it's clear we deal with an overlay file.

Reply:
Will do